quora-service/src/main/java/com/upgrad/quora/service/entity/UserEntity.java

```diff
@@ -27,19 +28,19 @@
 
     @Column(name = "uuid")
     @NotNull
-    private UUID uuid;
+    private String uuid;
```

Comment:
UUID can be datatype for ```uuid``` instead of String.

Reply:
if we will use the UUID as Datatype in the Users table then it will throw while we will retrieve data  from database as it store as string in the table. You can check the test case controller for the same.